examples/client/client.c

```diff
@@ -834,6 +834,7 @@ const char* starttlsCmd[6] = {
 static int StartTLS_Init(SOCKET_T* sockfd)
 {
     char tmpBuf[512];
+    char buf[3];     /* 3 digit number */
```

Comment:
The `buf` should still have a NULL terminator. Please make `buf[4]`. The XMEMCPY needs to be `XSTRLEN()+1`.
Also you still need to add spaces after the comma's `, ` for function arguments.